src/dstack/_internal/server/routers/volumes.py

```diff
@@ -66,12 +66,14 @@ async def get_volume(
 async def create_volume(
     body: CreateVolumeRequest,
     session: AsyncSession = Depends(get_session),
+    user: UserModel = Depends(Authenticated()),
     user_project: Tuple[UserModel, ProjectModel] = Depends(ProjectMember()),
 ) -> Volume:
     _, project = user_project
```

Comment:
No need to get user via Authenticated() since it's returned by ProjectMember(). Just do `user, project = user_project` like in other endpoints.

Reply:
fixed